content/en/docs/tasks/run-application/run-single-instance-stateful-application.md

```diff
@@ -176,7 +176,8 @@ specific to stateful apps:
 Delete the deployed objects by name:
 
 ```
-kubectl delete deployment,svc mysql
+kubectl delete deployment mysql
+kubectl delete svc mysql
```

Comment:
Rather than change the command, we could add a comment to explain what will happen. That is what I recommend.

Teaching readers that you delete multuple kinds at once is valuable; also, this is a task page rather than a tutorial.

Reply:
Maybe this page would be better off being placed in the tutorial section. If we did move it, we might reconsider.

Reply:
@sftim, I think we can mention these points on introductory tutorial pages, or we can have one introductory page in Kubernetes docs, that can introduce these analogies of Kubernetes to beginners.